specs/eventing/spec.md

```diff
@@ -309,11 +309,24 @@ for a `Subscription`.
+FilterSpec allows to define a filter. If multiple filters are specified, then
+the same semantics of `FilterSpec.All` is applied. If no filter dialect or empty
+object is specified, then the filter always accept the events.
+
+| Field Name   | Field Type          | Requirement | Description                                                                                                                                                                                                                                                                 | Constraints                                                                                                                                                                                                                                                          |
+| ------------ | ------------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `attributes` | `map[string]string` | Optional    | A map of context attribute names to values for filtering by equality. Each key in the map is compared with the equivalent key in the event context. An event passes the filter if all values are equal to the specified values. The value '' to indicate all strings match. | Nested context attributes are not supported as keys. Only string values are supported. Only exact matches will pass the filter.                                                                                                                                      |
+| `all`        | `[]FilterSpec`      | Optional    | All evaluates to true if all the nested expressions evaluate to true.                                                                                                                                                                                                       | All must contain at least one filter expression.                                                                                                                                                                                                                     |
```

Comment:
remove the leading "All" in the last column, it could be hard for some folks to parse. And for the 2nd sentence too.

Actually, the CE spec allows for an empty list so we should do so here too